### DISCUSSION TITLE
Support fonts etc in Theme Generator

### DISCUSSION THREAD

**Discussion by nileshtrivedi** [saadeghi/daisyui#2551] [Ideas / feature requests]
Currently the [theme generator tool](https://daisyui.com/theme-generator/) only supports randomization of colors.

It would be great if we can randomize font family, font-size, and variants/margin/padding of components to make them more or less condensed. Ideally, it should be done via a separate button to have a little more control on the theme.

Other potential enhancements would be customizations of gradient/image backgrounds etc. Similar to this Shadcn UI Theme generator: https://gradient.page/tools/shadcn-ui-theme-generator

**Comment by LuziferSenpai** [comment]
You can use https://www.realtimecolors.com/ for getting the right font, font-size etc.
